scripts/fetch-proto.sh

```diff
@@ -147,6 +154,8 @@ echo ""
 echo "Compiling gRPC files"
 generateGrpc "$PATH_ROOT/src/proto" "dapr/proto/common/v1/common.proto"
 generateGrpc "$PATH_ROOT/src/proto" "dapr/proto/internals/v1/apiversion.proto"
+# Also generate code for reminders to satisfy imports
+generateGrpc "$PATH_ROOT/src/proto" "dapr/proto/internals/v1/reminders.proto"
 generateGrpc "$PATH_ROOT/src/proto" "dapr/proto/internals/v1/service_invocation.proto"
```

Comment:
Same here - we shouldn't actually be using any of these except appcallback.proto, dapr.proto and common.proto as they're internal to the runtime and subject to change at any point.